packages/mobile/src/screens/sign-up-screen/screens/CreateEmailScreen.tsx

```diff
@@ -0,0 +1,214 @@
+import { SocialMediaLoginOptions } from '../components/SocialMediaLoginOptions'
+import { Heading, Page } from '../components/layout'
+import { Divider } from '../components/temp-harmony/Divider'
+import { Hint } from '../components/temp-harmony/Hint'
+import { IconAudiusLogoHorizontalColor } from '../components/temp-harmony/IconAudiusLogoHorizontalColor'
+import IconExclamation from '../components/temp-harmony/IconExclamation.svg'
+import type { SignUpScreenParamList } from '../types'
+
+const messages = {
+  title: 'Sign Up For Audius',
+  emailLabel: 'Email',
+  signUp: 'Sign Up Free',
+  haveAccount: 'Already have an account?',
+  signIn: 'Sign In',
+  subHeader: (
+    <>
+      Join the revolution in music streaming!{'\n'}Discover, connect, and create
+      on Audius.
+    </>
+  ),
+  socialsDividerText: 'Or, get started with one of your socials',
+  unknownError: 'Unknown error occurred.',
+  metaMaskNotRecommended: 'Signing up with MetaMask is not recommended.',
+  signUpMetamask: 'Sign Up With MetaMask',
+  learnMore: 'Learn More'
+}
+
+type SignUpEmailValues = {
+  email: string
+}
+
+export const CreateEmailScreen = () => {
+  const dispatch = useDispatch()
+  const navigation = useNavigation<SignUpScreenParamList>()
+  const existingEmailValue = useSelector(getEmailField)
+  const queryContext = useContext(AudiusQueryContext)
+  const initialValues = {
+    email: existingEmailValue.value ?? ''
+  }
+  const { color, cornerRadius } = useTheme()
+  const emailFormikSchema = useMemo(() => {
+    if (queryContext) {
+      return toFormikValidationSchema(emailSchema(queryContext))
+    }
+  }, [queryContext])
+
+  const handleSubmit = useCallback(
+    (values: SignUpEmailValues) => {
+      const { email } = values
+      // Set the email in the store
+      dispatch(setValueField('email', email))
+      navigation.navigate('CreatePassword', { email })
+    },
+    [dispatch, navigation]
+  )
+
+  return (
+    <Box h='100%'>
+      <Formik
+        initialValues={initialValues}
+        onSubmit={handleSubmit}
+        validationSchema={emailFormikSchema}
+      >
+        {({ handleSubmit, errors, values }) => (
+          <Page
+            style={css({
+              height: 'auto',
+              backgroundColor: color.background.white,
+              borderBottomLeftRadius: cornerRadius['2xl'],
+              borderBottomRightRadius: cornerRadius['2xl'],
+              zIndex: 2
+            })}
+          >
+            <Box alignSelf='center'>
+              <IconAudiusLogoHorizontalColor />
+            </Box>
+
+            <Heading
+              heading={messages.title}
+              description={messages.subHeader}
+              centered
+            />
+            <Flex direction='column' gap='l'>
+              {/* TODO: replace with harmony text input */}
+              <TextField name='email' label={messages.emailLabel} noGutter />
+              {errors.email === emailSchemaMessages.emailInUse ? (
```

Comment:
use formik's <ErrorMessage> component?

Reply:
Which component? This is our custom hint error thing